.github/workflows/estimate-review.yml

```diff
@@ -0,0 +1,54 @@
+name: Estimate-review
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+
+jobs:
+  estimate:
+    runs-on: ubuntu-latest
+    container:
+      image: golang:1.19
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          repository: 'redhat-appstudio/qe-tools'
+      - name: Setup Go environment
+        uses: actions/setup-go@v5
+
+      - name: Run estimate time needed for PR review
+        run: |
+          go run main.go estimate-review \
```

Comment:
@tnevrlka dumb question, why do we run the command like this rather than `./qe-tools ...`

Reply:
We'd need to build it first, wouldn't we? It's the same way in other GitHub workflows in this repository e.g. https://github.com/redhat-appstudio/qe-tools/blob/main/.github/workflows/coffee.yml#L22

Reply:
> I think we want to make sure we document this properly. Like how to use and enable. And more importantly the logic you implemented and the weighted scores you apply to files, etc.

On it. Do you think it deserves its own Jira story?

Reply:
Yeah I think it deserves it's own story. So if we have that tracker then we don't have to include it as part of this PR

Reply:
> We'd need to build it first, wouldn't we? It's the same way in other GitHub workflows in this repository e.g.
> 
> https://github.com/redhat-appstudio/qe-tools/blob/d0faf0c6459c0ab5404684ca7b0c4d25465847c3/.github/workflows/coffee.yml#L22

Ah ok, I didn't catch that in the other workflow. I think this might work for now but I would think we would want to build and run the command as `qe-tools` in "production"

Reply:
Fair enough, seems like the best practice